### PR TITLE
-adding id on the return object for add and update

### PR DESCRIPTION
**Pull request by prashantpawar**
Now `POST /candidates/` and `PUT /candidates/:id` request returns the updated object back.

### CODE REVIEW

File: server/routes/candidates.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ class Candidates {
 
     var newCandidate = req.body;
     db.add(candidateModel, newCandidate).then((result) => {
+      res.json(result)
```
Comment (jackofseattle) [inline server/routes/candidates.js]: missing semi colon - might wanna run style checker over this actually

File: server/routes/candidates.js [hunk 2]
```diff
@@ -68,6 +69,7 @@ class Candidates {
 
     db.update(candidateModel, req.params.id, updatedFields).then((result) => {
       if(result) {
+        res.json(result);
         res.status(204);
```
Comment (jackofseattle) [inline server/routes/candidates.js]: Any reason why you can't just return here and omit the else ?

File: server/helper/database.js [hunk 3]
```diff
@@ -17,40 +17,40 @@ class DbHelper {
...
     let flatModelSchema = flat.flatten(model.schema);
 
-    let key = model.getKey(id);
+    let key = model.getKey(obj.id);
     let indexKey = model.getKey("index");
 
-    multi.hset(indexKey, id, key);
+    multi.hset(indexKey, obj.id, key);
     for (let attributeName in flatModelSchema) {
       if(flatObj[attributeName]) {
```
Comment (jackofseattle) [inline server/helper/database.js]: So I can't set anything to an empty string, false, or 0?  Seems like weird behavior.

Comment (prashantpawar) [inline server/helper/database.js]: Fixed that

File: server/routes/candidates.js (hunk 1 above)
Comment (prashantpawar) [inline server/routes/candidates.js]: Done

**Comment by prashantpawar** [comment]
Merging it if it looks good

**Pull request merged**